src/Dapr.Actors/Runtime/ActorRuntime.cs

```diff
@@ -63,6 +66,26 @@ public void RegisterActor<TActor>(Func<ActorTypeInformation, ActorService> actor
 
             // Create ActorManagers, override existing entry if registered again.
             this.actorManagers[actorTypeInfo.ActorTypeName] = new ActorManager(actorService);
+            this.actorConfig.RegisteredActorTypes.Add(actorTypeInfo.ActorTypeName);
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        public void UseActorConfig(ActorConfig config)
```

Comment:
rename to UseActorSettings.
It would be better if  we can create ActorSettings and then pass it to user, user can then configure it by doing:
runtime.ConfigureActorSettings(Action<ActorSettings> configureActorSettings)

That way they dont need to worry about creating ActorSettings and just change what they want to change on ActorSettings object given to them.

Reply:
I'm not sure I understand the benefit here.  If the user doesn't specify anything, all properties are default.  If we were to pass them a prepopulated object, it would have all default (null, etc) values too.

Reply:
They don't need to new up the settings class.

Reply:
It seems pretty natural and normal to create an object like this.  Unless this is a convention I'd like to keep it this way.

Reply:
There is no convention, but when users create an object, they need to think about setting values for all of these but when an object is given, they just modify what they need to and assume default is already configured for it.

Reply:
Example:

```
            actorRuntime.ConfigureActorSettings(a =>
            {
                a.SetActorIdleTimeout(TimeSpan.FromSeconds(33));
                a.SetActorScanInterval(TimeSpan.FromSeconds(44));
                a.SetDrainOngoingCallTimeout(TimeSpan.FromSeconds(55));
                a.SetDrainRebalancedActors(true);
            });
```